qdrant-landing/content/blog/using-qdrant-and-langchain.md

```diff
@@ -0,0 +1,109 @@
+
+> Qdrant is one of the **top supported vector stores** on LangChain, with [extensive documentation](https://python.langchain.com/docs/integrations/vectorstores/qdrant) and [examples](https://python.langchain.com/docs/integrations/retrievers/self_query/qdrant_self_query).
+
+**How it Works:** LangChain receives a query, dispatches it to a vector database such as Qdrant, retrieves relevant documents, and then sends both the query and the retrieved documents into the large language model to generate an answer.
+
+![qdrant-langchain-rag](/blog/qdrant-and-langchain/flow-diagram.png)
+
+When supported by LangChain, Qdrant can help you set up effective question-answer systems, detection systems and chatbots that leverage RAG to its full potential. When it comes to long-term memory storage, developers can use LangChain to easily add relevant documents, chat history memory & rich user data to LLM app prompts via Qdrant. 
+
+## Common Use Cases
+
+Integrating Qdrant and LangChain can revolutionize your AI applications. Let's take a look at what this integration can do for you: 
+
+*Enhance Natural Language Processing (NLP):* 
+LangChain is great for developing question-answering **chatbots**, where Qdrant is used to contextualize and retrieve results for the LLM. We cover this in [our article](https://qdrant.tech/articles/langchain-integration/), and in OpenAI's [cookbook examples](https://cookbook.openai.com/examples/vector_databases/qdrant/qa_with_langchain_qdrant_and_openai) that use LangChain and GPT to process natural language.
+
+*Improve Recommendation Systems:*
+Food delivery services thrive on indecisive customers. The necessity for a multi-aim search process, where customers seek recommendations led to the development of [Recommendation API](https://qdrant.tech/documentation/concepts/explore/#recommendation-api), which is an alternative to regular vector search. With LangChain you can build systems for **e-commerce, content sharing, or even dating apps**.
+
+*Advance Data Analysis and Insights:* Sometimes you just want to browse results that are not necessarily closest, but still relevant. Qdrant's [Discovery API](https://qdrant.tech/articles/discovery-search/#discovery-search) is useful for **online stores**, where users are don't exactly know what they are looking for, but require constrained space in which a search is performed. 
+
+*Offer Content Similarity Analysis:* Ever been stuck seeing the same recommendations on your **music streaming service**? You may be held in a similarity bubble! As inputs get more complex, diversity becomes scarce, and it becomes harder to force the system to show something different. LangChain developers can use [Context API](https://qdrant.tech/articles/discovery-search/#context-search) for such cases.
```

Comment:
Unfortunately, none of these features is implemented in Langchain. Their interfaces for vector stores are rather simple, and search is the only endpoint we use so far.

I get why we want to put it in the article, but that might be a bit confusing. How about mentioning all these use cases, without pointing to our APIs directly?